arch/x86/kernel/cpu/resctrl/internal.h

```diff
@@ -54,6 +55,28 @@
 /* Max event bits supported */
 #define MAX_EVT_CONFIG_BITS		GENMASK(6, 0)
 
+/**
+ * cpumask_any_housekeeping() - Choose any CPU in @mask, preferring those that
+ *			        aren't marked nohz_full
+ * @mask:	The mask to pick a CPU from.
+ *
+ * Returns a CPU in @mask. If there are housekeeping CPUs that don't use
+ * nohz_full, these are preferred.
+ */
+static inline unsigned int cpumask_any_housekeeping(const struct cpumask *mask)
```

Comment:
this function at upstream was redone so looks little different.[(https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=a4846aaf39455fe69fce3522b385319383666eef)